### PR TITLE
added a choice in luciddream.md

### PR DESCRIPTION
**Pull request by critesjosh**
added a link to boil.md

### CODE REVIEW

**Comment by pmallory** [comment]
@casey-collab can you review this PR please?

File: english/lucidream/luciddream.md [hunk 1]
```diff
@@ -15,3 +15,5 @@ Yells and become super saiyan
 Flys to the moon
 
 [Turn into a potato](potato/potato.md)
+
+See what it feels like to [boil](boil/boil.md)
```
Comment (casey-collab) [inline english/lucidream/luciddream.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by critesjosh** [comment]
I was having trouble with the syntax used to create the link. I changed the option and I think I got it.

**Pull request closed without merging**